ti-pru-cgt-installer/version.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export custom_build=true
+export require_root=false
+
+function run_custom_build() {
+    mkdir -p "${builddir}/${package_name}_${deb_version}"
+    cp -r "${debcontroldir}"/* "${builddir}/${package_name}_${deb_version}"
+    (cd "${builddir}/${package_name}_${deb_version}" && debuild --no-lintian -i -us -uc -b)
```

Comment:
We run debuild in the main script. I'm trying to remove this "run_custom_build", you shouldn't use it if not 100% needed.